shipyard_blueprints/notion/shipyard_notion/cli/authtest.py

```diff
@@ -1,17 +1,24 @@
 import os
 import sys
-from shipyard_notion import NotionClient
+import requests
 
 
 def main():
-    try:
-        client = NotionClient(token=os.getenv("NOTION_ACCESS_TOKEN"))
-        c = client.connect()
```

Comment:
any reason why we the connect function wasn't updated?

Reply:
I just refactored the auth test because the connect method uses the `notion-client` package to establish a client, whereas the auth test simply uses requests. I chose this to not have to install `notion-client` for the root package [](edited-with-Axolo)